include/fiction/algorithms/simulation/sidb/sidb_simulation_parameters.hpp

```diff
@@ -5,14 +5,20 @@
 #ifndef FICTION_SIDB_SIMULATION_PARAMETERS_HPP
 #define FICTION_SIDB_SIMULATION_PARAMETERS_HPP
 
-#include "fiction/technology/physical_constants.hpp"
+#include "fiction/technology/physical_constants_and_ev_unit.hpp"
 
 #include <cassert>
 #include <cstdint>
 
+#include <units.h>
+
 namespace fiction
 {
 
+using units::literals::operator""_eV;
+using units::literals::operator""_nm;
+using units::literals::operator""_angstrom;
```

Comment:
Please add docstrings and RTD documentation.

Also, why are the unit literals defined in different places?